### DISCUSSION TITLE
Feature Request: Android Screensaver implementation

### DISCUSSION THREAD

**Discussion by redfox789** [Radiokot/photoprism-android-client#109] [Ideas]
Thanks for developing this app, I love it! 

One thing that would really take it over the top would be to integrate it with Android's screensaver function. This would allow photoprism users to set up a tablet or AndroidTV as a digital photo frame, a feature that is sorely missing at the moment for folks who have de-googled.

**Comment by Radiokot** [comment]
That's a nice idea, I'll consider adding this feature 👍🏻
